python/py-eyed3/Portfile

```diff
@@ -56,8 +57,12 @@ if {${name} ne ${subport}} {
     } else {
     # list here dependencies for Python != 2.7
         depends_lib-append \
-                    port:py${python.version}-deprecation \
-                    port:py${python.version}-filetype
+#            port:py${python.version}-coverage \
+            port:py${python.version}-deprecation
+#            port:py${python.version}-filetype \
+#            port:py${python.version}-packaging \
+#            port:py${python.version}-parsing \
+#            port:py${python.version}-toml
```

Comment:
as I said before, just remove the dependencies you initially added (only `deprecation` and `filetype`) are required and we should be good to go.